sdks/python/apache_beam/testing/util.py

```diff
@@ -75,6 +75,25 @@ def __repr__(self):
   return InAnyOrder(iterable)
 
 
+def equal_to_per_window(expected_dict_window_to_its_elements):
+  def matcher(elements):
+    actual_elements_in_window, window = elements
+    if window in expected_dict_window_to_its_elements:
+      expected_elements_in_window = list(
+          expected_dict_window_to_its_elements[window])
+      sorted_expected = sorted(expected_elements_in_window)
+      sorted_actual = sorted(actual_elements_in_window)
+      if sorted_expected != sorted_actual:
+        # Results for the same window don't necessarily come all
```

Comment:
<!--new_thread; commit:1d8257d6902c54d33062c23cce99e06fbf42c41d; resolved:0-->
Add a TODO here with a link to an appropriate issue.

Reply:
That is the expected behavior when there are additional firings.